content/learn/userguides/secondary-analysis/10x.md

```diff
@@ -36,10 +39,32 @@ Cell barcode correction uses as input files in BAM file format; thus the fastq f
 
 The [Attach10xBarcodes](https://github.com/HumanCellAtlas/sctools) tool is then used to attach 10X barcodes found in the fastq sample to the corresponding reads in the BAM file, producing a barcoded.bam file. The tool uses as input three files:1) the original fastq file of forward reads (r1), containing the unique molecular identifier and cell barcode sequences, 2) an index fastq file which contains the Illumina sample barcodes, and 3) the unmapped bam file of reverse reads (r2), which is the alignable genomic information. The reads in these files are in the same order, and the tool works sequentially through the data, putting the barcodes found for each read of the first 2 fastq files  onto the corresponding read of the third (BAM) file.  The program returns as output a barcoded.bam file containing the reads with correct barcodes (this includes barcodes that came within one error of matching, which are corrected by this tool). These are assigned a “CB” tag. Uncorrected barcodes are preserved and given a “CR” (Cell barcode Raw) tag. During this process, barcoded reads are sorted into cell groups so that the data is divided into reads per cell chunks.
 
+The remainder of the pipeline processes these chunks in parallel.
+
+TODO: make the rest of the docs have links like this?
+
+WDL: [Attach10xBarcodes.wdl](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/Attach10xBarcodes.wdl)
+
+Docker: [Python3-scientific](https://github.com/HumanCellAtlas/skylab/blob/master/docker/python3-scientific/Dockerfile)
+
+Key Library: [Sctools](https://github.com/HumanCellAtlas/sctools)
+
 ## Alignment
 
-The [STAR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/) alignment software is then used to map the reads to the genome, using the GRCh38 human genome primary sequence as reference. STAR (Spliced Transcripts Alignment to a Reference) is widely-used for RNA-seq alignment and generates transcript count information as well as gene identification assignments. Alignment to the V27 GenCode transcriptomic reference is also performed using the [HISAT2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655817/) software to ensure that actual biotypes (and not pseudogenes, for example) are identified.
+The [STAR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/) alignment software is used to map the reads to the GRCh38 human genome primary assembly reference. STAR (Spliced Transcripts Alignment to a Reference) is widely-used for RNA-seq alignment and identifies the best matching location(s) on the reference for each sequencing read.
+
+## Gene Annotation
+
+The [TagGeneExon]() tool from [DropSeqTools] is used to annotate each read with the type of sequence it aligned to. These annotations include `INTERGENIC`, `INTRONIC`, and `EXONIC`, and are stored in the `XF` tag. In cases where the gene corresponds to an intron or exon, the name of the gene that overlaps the alignment is associated with the read and stored in the `GE` tag.
```

Comment:
Can you point me to the right citation is for the tools? I don't see anything obvious other than reference the McCarroll lab website.